indicators/hyphisher-hypixel-e393965e.yml

```diff
@@ -1,40 +1,39 @@
-title: Hyphisher Hypixel Phishing Kit e393965e
+title: Hyphisher Phishing Kit e393965e
 description: |
-  Hypixel phishing kit that works by socially engineering
-  victims into thinking they have been gifted an item from
-  the Hypixel store, it then asks the victim to enter their
-  username and email, upon doing this the backend will send
-  the victim an email asking to verify the gift by entering
-  their authentication code into the website.
+  `Hyphisher` is a phishing kit that focuses on targeting 
+  Hypixel & LunarClient users using various lures such 
+  as free ranks, cosmetics or modifications. 
+  
+  It works by socially engineering victims into thinking they 
+  have been gifted an item from the attacker/compromised account, 
+  it then asks the victim to enter their username and email, upon 
+  doing this the backend will send the victim an email asking to 
+  verify the gift by entering their authentication code into the 
+  website.
   
   This specific kit dubbed `Hyphisher` is developed by a threat
-  actor named `GuteNacht`
+  actor named `GuteNacht` and has since been modified and forked
+  by other threat actors within the threat landscape.
   
 references:
   - https://urlscan.io/result/e393965e-1285-489d-b9db-a57aa1cca8cc
+  - https://urlscan.io/result/87946bb2-475d-4a89-ab1f-b1bd124736e5
 
 detection:
-
-  processingScript:
-    requests|contains: 'handleUserInfo.js'
     
   siteElementScript:
     requests|contains: 'site.js'
     
   jsCodeSnippets:
     js|contains|all:
       - 'sendCode'
-      - 'useUsername'
       - 'sendEmail'
-      - 'processForm'
-      - 'clearWaitingOverlay'
-      - 'copyToClipboard'
-      - 'getCookie'
-      - 'unserialize'
 
-  condition: processingScript and siteElementScript and jsCodeSnippets
+
+  condition: siteElementScript and jsCodeSnippets
 
 tags:
   - kit
   - target.hypixel
+  - target.lunar_client
```

Comment:
Minor suggestion (fine to ignore this though): if this is a completely different target, would it be worth creating a new rule to track them separately?

Reply:
The core functionality of the kit itself is the same the only thing that changes is the website they try to imitate, they've used several lures such as Hypixel, Lunar Client, Skycrypt & Essential where the latter two are Minecraft modifications for Hypixel Skyblock